x-pack/legacy/plugins/maps/public/layers/styles/vector/properties/dynamic_color_property.test.js

```diff
@@ -260,56 +260,77 @@ describe('get mapbox color expression', () => {
       test('should return null when color ramp is not provided', async () => {
         const dynamicStyleOptions = {
           type: COLOR_MAP_TYPE.ORDINAL,
-          field: {
-            name: 'myField',
-          },
         };
         const colorProperty = makeProperty(dynamicStyleOptions);
         expect(colorProperty._getMbColor()).toBeNull();
       });
 
-      test('should return mapbox expression for color ramp', async () => {
-        const dynamicStyleOptions = {
-          type: COLOR_MAP_TYPE.ORDINAL,
-          field: {
-            name: 'myField',
-          },
+      test('Should set the correct mapbox style', async () => {
```

Comment:
Why test description change? "Should set the correct mapbox style" is pretty vague. "should return mapbox expression for color ramp" is more specific in what the test is verifying